Podcast/InternalProfileViewController.swift

```diff
@@ -133,10 +131,6 @@ class InternalProfileViewController: ViewController, UITableViewDelegate, UITabl
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-    }
-    
-    override func viewDidAppear(_ animated: Bool) {
-        super.viewDidAppear(animated)
         fetchSubscriptions()
```

Comment:
I think it'd be better to fetch stuff and do API requests in viewDidAppear, WillAppear is usually for view hiding, moving, etc. I've read this somewhere else but I can't find it but here's a SO post that reiterates https://stackoverflow.com/questions/5562938/looking-to-understand-the-ios-uiviewcontroller-lifecycle

Reply:
got it will change to be in viewDidAppear